src/main/resources/Mobs.json

```diff
@@ -1,7 +1,22 @@
-{
-  "Mob":{
-    "Name": "",
-    "Combat Lvl": 2,
-    "Drops":[""]
+[
+  {
+    "name": "Goblin",
+    "combatLvl": 2,
+    "drops":["bar", "foo"]
   },
-}
+  {
+    "name": "Imp",
+    "combatLvl": 33,
+    "drops":["flour", "bucket"]
+  },
+  {
+    "name": "chicken",
+    "combatLvl": 99,
+    "drops":["feather"]
+  },
+  {
+    "name": "yew tree",
```

Comment:
Why is a yew tree a mob?

Reply:
I was testing some stuff and forgot to delete it